journal/updated/01-journal.adoc

```diff
@@ -386,7 +386,12 @@ and Adam gave names to them.
 And God took him, and put him in the garden of Eden, to dress it and to keep it.
 And the Lord commanded the man, saying,
 '`Of every tree of the garden thou may freely eat,`'`" Gen 2:15-16.
-This is the first Thou spoken to man
+This is the first Thou spoken to man footnote:[Richard Davies is not
```

Comment:
because there is no punctuation after the word man, this footnote needs a caret ^ 
It should read like this: 

```
This is the first Thou spoken to man^ 
footnote:[Richard Davies is not........]
```